### PR TITLE
Merge branch 'release_v0.12' into develop_v0.12

### PR DESCRIPTION
**Pull request by saliHaguli**

### CODE REVIEW

File: spotinst/mrscaler_aws_terminationPolicies/fields_spotinst_mrscaler_aws_terminationPolicies.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
+package mrscaler_aws_terminationPolicies
+
+import (
+	"fmt"
```
Comment (liranp) [inline spotinst/mrscaler_aws_terminationPolicies/fields_spotinst_mrscaler_aws_terminationPolicies.go:4]: Please execute `goimports`.

File: spotinst/resource_spotinst_mrscaler_aws.go [hunk 2]
```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/terraform-providers/terraform-provider-spotinst/spotinst/mrscaler_aws_terminationPolicies"
```
Comment (liranp) [inline spotinst/resource_spotinst_mrscaler_aws.go:10]: Also here.

File: website/docs/r/mrscaler_aws.html.markdown [hunk 3]
```diff
@@ -431,6 +431,64 @@ Each `*_scaling_*_policy` supports the following:
...
+  }
+```
+
+## Argument Reference
+
+* `termination_policies` - (Optional) Allows defining termination policies for EMR clusters based on CloudWatch Metrics.
+* `statements` - (Required) 
+* `namespace` - (Required) Must contain the value: `AWS/ElasticMapReduce`.
+* `metric_name` - (Required) The name of the metric in CloudWatch which the statement will be based on.
+                             Example: `AppsRunning`.
```
Comment (liranp) [inline website/docs/r/mrscaler_aws.html.markdown:483]: Why did you add a new line for the example? We don't seem to use it in the other [examples](https://github.com/spotinst/terraform-provider-spotinst/blob/release_v0.12/website/docs/r/ocean_aws.html.markdown).

File: website/docs/r/mrscaler_aws.html.markdown [hunk 4]
```diff
@@ -431,6 +431,64 @@ Each `*_scaling_*_policy` supports the following:
...
+* `statistic` - (Optional, Default: `sum`) The aggregation method of the given metric. Valid Values: `average` | `sum` | `sampleCount` | `maximum` | `minimum`                 
+* `unit` - (Optional, Default: `count`) The unit for a given metric. Valid Values: `seconds` | `microseconds` | `milliseconds` | `bytes` | `kilobytes` | `megabytes` | `gigabytes` | `terabytes` | `bits` | `kilobits` | `megabits` | `gigabits` | `terabits` | `percent` | `count` | `bytes/second` | `kilobytes/second` | `megabytes/second` | `gigabytes/second` | `terabytes/second` | `bits/second` | `kilobits/second` | `megabits/second` | `gigabits/second` | `terabits/second` | `count/second` | `none`                     
+* `threshold` - (Required) The value that the specified statistic is compared to.
+                           Example: `2.0`.
```
Comment (liranp) [inline website/docs/r/mrscaler_aws.html.markdown:487]: Also here.

File: website/docs/r/ocean_aws.html.markdown [hunk 5]
```diff
@@ -118,46 +83,20 @@ whitelist = ["t1.micro", "m1.small"]
...
-    name = "balancer-name"
-    type = "CLASSIC"
-  }
-```
-
+* `tags` - (Optional) Optionally adds tags to instances launched in an Ocean cluster.
+* `key` - (Optional) The tag key.
+* `value` - (Optional) The tag value.
 * `fallback_to_ondemand` - (Optional, Default: `true`) If not Spot instance markets are available, enable Ocean to launch On-Demand instances instead.
 * `utilize_reserved_instances` - (Optional, Default `true`) If Reserved instances exist, OCean will utilize them before launching Spot instances.
```
Comment (liranp) [inline website/docs/r/ocean_aws.html.markdown:94]:
```suggestion
* `utilize_reserved_instances` - (Optional, Default `true`) If Reserved instances exist, Ocean will utilize them before launching Spot instances.
```

File: spotinst/mrscaler_aws_terminationPolicies/fields_spotinst_mrscaler_aws_terminationPolicies.go (hunk 1 above)
Comment (saliHaguli) [reply spotinst/mrscaler_aws_terminationPolicies/fields_spotinst_mrscaler_aws_terminationPolicies.go:4]: fixed and pushed

File: spotinst/resource_spotinst_mrscaler_aws.go (hunk 2 above)
Comment (saliHaguli) [reply spotinst/resource_spotinst_mrscaler_aws.go]: fixed

File: website/docs/r/mrscaler_aws.html.markdown (hunk 3 above)
Comment (saliHaguli) [reply website/docs/r/mrscaler_aws.html.markdown]: fixed
